chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java

```diff
@@ -456,7 +457,7 @@ public void onSucc(@Nullable final Conversation conversation) {
                 }
 
                 if (conversation == null) {
-                    callback.onFail("Cannot find the conversation");
+                    callback.onFail(new ConversationNotFoundError(conversation.getId()));
```

Comment:
`if (conversation == null)` we should not be able to getId from the conversation?

Reply:
Oh my bad :(